superbuild/projects_modules/openssl.cmake

```diff
@@ -92,7 +92,7 @@ else (WIN32)
     UPDATE_COMMAND ""
     PATCH_COMMAND ""
     DEPENDS ${${ep}_dependencies}
-    CONFIGURE_COMMAND ${EP_PATH_SOURCE}/${ep}/config no-zlib shared --prefix=${build_path} --openssldir=${build_path} 
+    CONFIGURE_COMMAND ${EP_PATH_SOURCE}/${ep}/config no-zlib  shared --prefix=${build_path} --openssldir=${build_path} 
```

Comment:
to can remove the additional space here

Reply:
Sorry but I'am checking mac os package gener now. I'am blocked.